### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by pratikkorat26** [hunglc007/tensorflow-yolov4-tflite#281]
Train.py
Hey what's conv in the loss function can someone 
explain me 

And in training why he has split the output into the pred and con

**Comment by guysoft** [comment]
Its prediction and confidence, YOLOv4's model output is 3 things:
* The location of the box (ancor) which is x,y,width,height
* The confidence that there is an object
* The objects ids in a list
See the head section here: https://becominghuman.ai/explaining-yolov4-a-one-stage-detector-cdac0826cbd7

This issue can be closed.
